simple_history/models.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         verbose_name=None,
         bases=(models.Model,),
         user_related_name="+",
+        user_db_constraint=True,
```

Comment:
Last comment: can you move the argument to the bottom? In v3 we'll look into updating how args are passed, but for now, it's feasible that someone could currently be passing args as positional args and this could break for them by changing the order.